.github/workflows/CI.yml

```diff
@@ -43,7 +43,7 @@ env:
   WIN_SIGNED_PKG_FOLDER: OTelCollectorAuthenticode/AuthenticodeSigner-SHA256-RSA
 
 concurrency:
-  group: ci
+  group: ci-${{ github.ref_name }}
```

Comment:
Interesting with the use of ref_name

Reply:
It's just so the different branches don't cancel each other when running the CI at the same time.

Reply:
True. Also can be used with tags as well so interesting use cases.

Reply:
Yep, but as of now, we only run the CI on push to `main`, `dev`, or `release/v*`.